pyproject.toml

```
@@ -0,0 +1,96 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "PyPDF2"
+authors = [{ name = "Mathieu Fenniak", email = "biziqe@mathieu.fenniak.net" }]
+maintainers = [{ name = "Martin Thoma", email = "info@martin-thoma.de" }]
+description = "A pure-python PDF library capable of splitting, merging, cropping, and transforming PDF files"
+readme = "README.md"
+dynamic = ["version"]
+license = "BSD-3-Clause"
```

```suggestion
license = { file = "LICENSE" }
```

I'm very sorry about this back-and-forth. I know looked it up:

* https://flit.pypa.io/en/latest/pyproject_toml.html#metadata-section
* https://peps.python.org/pep-0621/#license
* https://python-poetry.org/docs/pyproject/#license

So you can see that poetry and flit define different types of values for this field. PEP 621 is not clear how it should be structured.

I've opened a question about this on PyPI discussions https://github.com/pypi/warehouse/discussions/12407

As flit does not accept a string-entry in here, we have to change it back to what you originally had.